py3/bluesky-0.4.3/meta.yaml

```diff
@@ -40,9 +40,13 @@ test:
     - bluesky
     - bluesky.broker_callbacks
     - bluesky.broker_examples
+    - bluesky.callbacks
+    - bluesky.callbacks.core
+    - bluesky.callbacks.olog
```

Comment:
I suspect that this will fail if you do not have olog configure properly.  If it does not find the configuration file it is looking for it will _on import_ use raw_input to ask for information.  This is why we do not have any import tests in the pyOlog recipe.